Add a UI element to cycle through nick completions on mobile

astorije opened the pull request:
This solves #339.

This was heavily inspired by https://github.com/maxpoulin64/lounge/commit/a877e46.
Clearly not a definitive solution but a good easy start to have and to improve later on.

### Result

![nick-cycler](https://cloud.githubusercontent.com/assets/113730/19545872/08d50b0e-9658-11e6-9808-4c5efc1cd78b.gif)

maxpoulin64 commented on client/css/style.css:
```diff
@@ -1731,6 +1740,11 @@ button {
 		transition: none;
 	}
+
+	/* Nick cycler should only be displayed on mobile */
```
Actually, I don't really see any harm into having it available on desktop as well. Makes it easy to discover for people that are not used to the tab mechanic.

astorije commented:
@maxpoulin64, made some changes regarding your comment:

- Rebased with `master`
- Do not hide on mobile anymore
- Consequence of this:
  - Use a `click` event now
  - Added a tooltip
  - Re-focus after clicking the button

Mind confirming/updating your review?

williamboman commented:
Loses input focus on mobile. Perhaps bypass the mobile detection and just `input.focus()`?

astorije commented:
> Loses input focus on mobile. Perhaps bypass the mobile detection and just `input.focus()`?

Same issue than #483.
Actual fix for that issue should fix both at the same time.
